DevelopGitHub/GitHubExtension/GitHubExtension.Activity.External.WebAPI/Services/EventParserHelper.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections.Generic;
+using GitHubExtension.Activity.External.WebAPI.EventParsers;
+using GitHubExtension.Activity.External.WebAPI.Exceptions;
+using Newtonsoft.Json.Linq;
+
+namespace GitHubExtension.Activity.External.WebAPI.Services
+{
+    public static class EventParserHelper
+    {
+        #region Event type names
+        public const string CommitCommentEvent = "CommitCommentEvent";
+        public const string CreateEvent = "CreateEvent";
+        public const string DeleteEvent = "DeleteEvent";
+        public const string ForkEvent = "ForkEvent";
+        public const string GolumEvent = "GolumEvent";
+        public const string IssueCommentEvent = "IssueCommentEvent";
+        public const string IssuesEvent = "IssuesEvent";
+        public const string MemeberEvent = "MemeberEvent";
+        public const string PullRequestEvent = "PullRequestEvent";
+        public const string PullRequestReviewCommentEvent = "PullRequestReviewCommentEvent";
+        public const string PushEvent = "PushEvent";
+        public const string RealiseEvent = "RealiseEvent";
+        public const string WatchEvent = "WatchEvent";
+        #endregion
+
+        private static readonly Dictionary<string, BaseEventParser> Parsers = new Dictionary<string, BaseEventParser>()
+        {
+            //Registering parsers for event types
+            { CommitCommentEvent, new CommitCommentEventParser()},
```

Comment:
I thought that it will be too many models for each event type. Each event has an field called payload that is different between each event type.